modules/__tests__/mail-helpers.test.js

```diff
@@ -0,0 +1,96 @@
+/* eslint-env jest */
+'use strict';
+const nodemailer = require('nodemailer');
+const { getSendAddress, buildMailOptions } = require('../mail-helpers');
+
+describe('getSendAddress', () => {
+    const expectedDefault = `noreply@biglotteryfund.org.uk`;
+    const expectedInternal = `noreply@blf.digital`;
+
+    it('it should return default send from address for external send to addresses', () => {
+        expect(getSendAddress('example@example.com')).toBe(expectedDefault);
+        expect(getSendAddress('example@gmail.com')).toBe(expectedDefault);
+    });
+
+    it('should return internal send from address for internal send to addresses', () => {
+        expect(getSendAddress('example@biglotteryfund.org.uk')).toBe(expectedInternal);
+        // Assert against similar looking but incorrect emails to test for false positives
+        expect(getSendAddress('example@biggerlotteryfund.org.uk')).toBe(expectedDefault);
+        expect(getSendAddress('example@biggestlotteryfund.org.uk')).toBe(expectedDefault);
+        expect(getSendAddress('biglotteryfund.org.uk@example.com')).toBe(expectedDefault);
+    });
+});
+
+async function mockEmail(mailOptions) {
+    const transport = nodemailer.createTransport({
+        streamTransport: true,
+        newline: 'unix',
+        buffer: true
+    });
+
+    const info = await transport.sendMail(mailOptions);
+    // Remove generated properties from message
+    info.message = info.message
```

Comment:
@mattandrews Switched to using snapshot tests as it allows us to assert against a real email response. Would appreciate thoughts. Particularly around this hackery which removes generated content from the responses so we can consider the snapshots static.